docs/configuration.md

```diff
@@ -26,13 +26,69 @@ variables.
 - Applies to: listener, worker, cron.
 - Format: string.
 
+###  CONFIG_REDIS_USERNAME
+
+- Redis ACL user name
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: string.
+
+###  CONFIG_REDIS_PASSWORD
+
+- Redis ACL password
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: string.
+
+###  CONFIG_REDIS_SSL
+
+- Whether use SSL to connect to Redis
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: true or false.
+
+###  CONFIG_REDIS_CA_FILE
+
+- Certification authority to validate Redis server TLS connections with
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: path to file as string.
+
+###  CONFIG_REDIS_CERT
+
+- The path to the client SSL certificate
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: path to file as string.
+
+###  CONFIG_REDIS_PRIVATE_KEY
+
+- The path to the client SSL private key
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: path to file as string.
+
 ###  CONFIG_REDIS_SENTINEL_HOSTS
 
 - URL of Redis sentinels.
 - Optional. Required only when using a Redis cluster with sentinels.
 - Applies to: listener, worker, cron.
 - Format: list of URLs separated by ",".
 
+### CONFIG_REDIS_SENTINEL_USERNAME
```

Comment:
We can't use CONFIG_REDIS_USERNAME?

Reply:
No because ACL for sentinels can be different to ACL for master. See https://github.com/redis/redis-rb/issues/1213.

Reply:
outrageous!